src/PhpImap/Mailbox.php

```diff
@@ -793,7 +793,41 @@ public function getMailboxes($search = "*") {
 		}
 		return $arr;
 	}
-
+    /**
+     * Get folders list
```

Comment:
This library uses `tab` for indention, not spaces. Please fix your code formatting,